common.yml

```diff
@@ -11,9 +11,9 @@ services:
       - PFB_DB_PORT=5432
       - PFB_DB_USER=pfb
       - PFB_SECRET_KEY=secret
-      - PFB_AWS_BATCH_ANALYSIS_JOB_QUEUE_NAME
-      - PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME_REVISION
-      - PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME
+      - PFB_AWS_BATCH_ANALYSIS_JOB_QUEUE_NAME=dummy-test-pfb-analysis-job-queue
```

Comment:
I assume there is dotenv for different envs when deploying and it gets pulled and used. These are here so that running analysis works locally as well. Sounds right?

Reply:
Yeah, they're defined in the `tfvars` files and loaded into the environment for the deployed tasks via the task configuration files in `deployment/terraform/task-definitions/`.